Adding first input to global index pages documentation

MayaBerd opened the pull request.

MayaBerd committed:
Adding first input to global index pages documentation

MayaBerd committed:
Moving home page FAQs to overall FAQs

MayaBerd committed:
updating global index pages documentation

MayaBerd committed:
time and cost report changes

MayaBerd committed:
Merge branch 'dev' into Global-indes-pages-documentation

marcalcobe commented on docs/user-guide/home/global-modules/README.md:
```diff
@@ -0,0 +1,134 @@
+
+Global modules present an overview of the projects that you are a member of or have permissions to see. Here you will find a summary of all the entries in the respective modules across your projects, such as **Activity, Work packages, Calendars, Boards**, etc.
+
+To find **Global modules** menu simply click on the logo in the header of the application, it will be displayed on the left side. 
```
@MayaBerd maybe we should add also that you can access the global modules from the top right grid menu icon.

MayaBerd replied:
Excellent suggestion, I added it. Thank you.

marcalcobe commented on docs/user-guide/home/global-modules/README.md:
```diff
@@ -0,0 +1,134 @@
+
+On the left you will have the following options:
+
+**All projects** is the default view and shows only active projects. If you do not have administrator rights, this view will be the same as **My projects** for you.
```
The part of administration rights and being the same as My projects is not true. This menu entry will contain both projects that you are member of and Public projects independent of your rights on them.

marcalcobe replied:
It is true though that only active projects are displayed and archived projects are not.

marcalcobe commented on docs/user-guide/home/global-modules/README.md:
```diff
@@ -0,0 +1,134 @@
+
+**All projects** is the default view and shows only active projects. If you do not have administrator rights, this view will be the same as **My projects** for you.
+
+**My projects** is the view that will show projects you are a member of and public projects. When this menu entry is selected, an **I am member** filter will be activated under Filters. You can then add various other filters to adjust the view.
```
I don't think this is correct. In the My projects section the users will only see the projects where they are members, not the public projects.

MayaBerd replied:
yes, you are right. removed the part on public projects.

marcalcobe commented on docs/user-guide/home/global-modules/README.md:
```diff
@@ -0,0 +1,134 @@
+
+**Work packages** global module will show a list of all the work packages from the projects you are a member of and all public projects. You can select your **Favorite** and **Default** work package filters in the left side menu.
+
+PUBLIC - ???
```
The work package module work the same as inside a project. You could have work package table views marked as Public or Private. Note that if a view is marked as Private or Public but also Favourite they will be shown in the Favourite section.

MayaBerd replied:
adding this.

marcalcobe commented on docs/user-guide/home/global-modules/README.md:
```diff
@@ -0,0 +1,134 @@
+
+## Work packages
+
+**Work packages** global module will show a list of all the work packages from the projects you are a member of and all public projects. You can select your **Favorite** and **Default** work package filters in the left side menu.
```
I think we should change list for table.

marcalcobe commented on docs/user-guide/home/global-modules/README.md:
```diff
@@ -0,0 +1,134 @@
+
+**News** global module will display all news that have been published in projects you are a member of, have administrative privileges to see and public projects. 
+
+TIMEFRAME - ?
```
As far as I know there is no timeframe for news, it is a paginated view until the very first new created.

marcalcobe commented on docs/user-guide/home/global-modules/README.md:
```diff
@@ -0,0 +1,134 @@
+
+## Meetings
+
+**Meetings** global module will provide a list of all upcoming an past meetings you have created, been invited to, attended or have administrative privileges to see. The default view will show the **Upcoming meetings**. The most current meetings are shown on top of the list. 
```
I don't think "the most current meetings" is clear, maybe "the closest meetings"?

psatyal replied:
I changed this in my commit to: "The default view will show the **Upcoming meetings**, in chronological order (earliest first)."

marcalcobe commented on docs/user-guide/home/global-modules/README.md:
```diff
@@ -0,0 +1,134 @@
+
+You can also choose to see all **Past meetings** or apply one of the filters based on your **Involvement** in a meeting. 
+
+You can create a new meeting directly from within the global modules menu by clicking a green ***Meeting** button either in the top right corner at the bottom of the sidebar. 
```
"+Meeting" instead of "*Meeting"

psatyal replied:
Fixed in my commit.

marcalcobe reviewed (commented):
I've added some comments with my feedback.

psatyal committed:
Changes to the global index docs and FAQ

- Mostly minor changes in phrasing/grammar
- Fixed image links
- Added alt text where missing

psatyal reviewed (commented):
I made some changes to in a later commit (mostly phrasing). Most of @marcalcobe's comments seem to still apply, however. I've resolved the ones I address in my commit :)

MayaBerd committed:
Update README.md

MayaBerd committed:
Update README.md

MayaBerd committed:
adding changes

MayaBerd committed:
Merge branch 'dev' into Global-indes-pages-documentation
